moveit_core/robot_state/include/moveit/robot_state/attached_body.h

```diff
@@ -110,6 +111,47 @@ class AttachedBody
     return attach_trans_;
   }
 
+  /** \brief Get the fixed transform to a named frame on this body (not a transform to a visual or collision shape) */
+  const Eigen::Affine3d& getFixedTransform(const std::string& frame_name) const
+  {
+    return getNamedTransform(frame_name);
+  }
+
+  /** \brief Get the fixed transform to a named frame on this body (not a transform to a visual or collision shape) */
+  const Eigen::Affine3d& getNamedTransform(const std::string& frame_name) const
+  {
+    return named_frames_.find(frame_name)->second;
+  }
+
+  /** \brief Returns true if the named
+   */
+  bool hasNamedTransform(const std::string& frame_name) const
+  {
+    return (named_frames_.count(frame_name) > 0);
```

Comment:
Better use `find(frame_name) != end()`. `count()` needs to process all elements.

Reply:
Thanks, very helpful